deployments/compose/README.md

```diff
@@ -13,6 +13,8 @@ That's it! To check the status of the containers:
 docker compose ps
 ```
 
+To access the lakeFS UI, go to http://localhost:8000. The setup stage is done for you, and your user name and passwords are AKIAIOSFODNN7EXAMPLE and wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY compatibly. 
```

Comment:
```suggestion
To access the lakeFS UI, go to http://localhost:8000. The docker-compose also runs the setup step and configure admin user with the following credentials:

Access key ID: `AKIAIOSFODNN7EXAMPLE`
Secret access key: `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY` 
```

Reply:
done

Reply:
UI uses the terms 'access key id' and 'secret access key' like S3.

Reply:
thanks, fixed.